### PR TITLE
Create ticket for end user

### PR DESCRIPTION
**Pull request by tmcdon89**
This allows the selection of ticket owner thereby allowing phone
tickets to be entered.
This uses the select2 query plugin to make ajax calls to the database
for search criteria.

Its important to edit the searched columns in the
“UserSearchController.php” file

### CODE REVIEW

File: src/Controllers/UserSearchController.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Kordy\Ticketit\Controllers;
+
+use App\Http\Controllers\Controller;
+use Input;
```
Comment (thekordy) [inline src/Controllers/UserSearchController.php]: I had to change `use Input;` to `use Illuminate\Support\Facades\Input;` in order to work .. other than that the ajax gets back an error as follow:

``` html
FatalErrorException in UserSearchController.php line 13: 
Class 'Input' not found
```

Although, I tried using the Request `Illuminate\Http\Request` and it worked just fine too

``` php
use Illuminate\Http\Request;

public function userSearch(Request $request) {

        $keyword = $request->get('q');
```

**Comment by thekordy** [comment]
This is really good feature and yet simple, I like your work. But I think it would be more usable and easier if we could make the search works on the default fields by default, and we may work on configurable custom fields in future.

**Comment by tmcdon89** [comment]
Thanks for the feedback. I actually meant to comment out the search fields that aren't default just to show them as examples. I agree they shouldn't be installed by default. 

As to using request that makes a lot more sense and I'll update shortly. I get stuck with bad old habits sometimes!

**Comment by thekordy** [comment]
> I get stuck with bad old habits sometimes!

Who doesn't! :)

**Pull request merged**